classes/forms.php

```diff
@@ -411,15 +411,11 @@ protected function force_string( $val ){
 	 */
 	protected static function add_details( $forms ){
 	    if( empty( $forms ) ){
-	        return [];
+			return [];
         }
 
-        if( ! empty( $valid_forms = get_transient( self::$registry_cache_key ) ) ) {
-            return $valid_forms;
-        }else{
-            $valid_forms = [];
-        }
-		
+		$valid_forms = [];
```

Comment:
In the PR you callout intentionally bypassing the cache, but once merged that decision will be buried, so it may be helpful to add a comment to explain the intent of bypassing the cache.

Reply:
Good point. Added in https://github.com/CalderaWP/Caldera-Forms/commit/1a6a1e0c69fa2152df0e64e5ac8dc2c8e937ba88